src/app/app.module.ts

```diff
@@ -26,6 +28,8 @@ import { Widget3Component } from './widgets/widget3/widget3.component';
   imports: [
     BrowserModule,
     AppRoutingModule,
+    GuiElementsModule,
+    PlaygroundModule,
     GridsterModule
```

Comment:
I would keep imports sorted alphabetically.

Reply:
> I would keep imports sorted alphabetically.

Since this is done automatically when creating a component, this might be a pain to maintain. We could add linting rules: https://eslint.org/docs/latest/rules/sort-imports